cmd/maya-exporter/main.go

```diff
@@ -23,6 +24,7 @@ func run() error {
 	// Create & execute new command
 	cmd, err := command.NewCmdVolumeExporter()
 	if err != nil {
+		log.Println("Can't execute the command, found err :", err)
```

Comment:
let us be elaborate here. Which command?

Reply:
@AmitKumarDas there err will contain the details about the command or flags or any other error in the hierarchy.

Reply:
ok